### PR TITLE
Overhaul sound lists to introduce aliases, rethink editing behavior

### PR DESCRIPTION
**Pull request by Crystalwarrior**
You will no longer overwrite the preexisting sondlists, what happens instead is you can play a custom sfx based on your input in the box. It will clear if sticky sfx are off.

Aliases are functional as well,
`filename = alias`

Old behavior with just filenames still works.

### CODE REVIEW

**Review by in1tiate — APPROVED** [review]
I like this behavior more, it's closer to the way iniswaps now work.

File: src/courtroom.cpp [hunk 1]
```diff
@@ -4032,28 +4030,32 @@ void Courtroom::set_sfx_dropdown()
...
-  if (soundlist.size() <= 0) {
-    soundlist = ao_app->get_list_file(
+  if (sound_list.size() <= 0) {
+    sound_list = ao_app->get_list_file(
         ao_app->get_theme_path("character_soundlist.ini"));
-    if (soundlist.size() <= 0) {
-      soundlist = ao_app->get_list_file(
+    if (sound_list.size() <= 0) {
+      sound_list = ao_app->get_list_file(
           ao_app->get_default_theme_path("character_soundlist.ini"));
```
Comment (oldmud0) [inline src/courtroom.cpp:4041]: Why do we store this in the theme folder instead of the base folder like the user-defined iniswaps INI?

**Review by oldmud0 — APPROVED** [review]

File: src/courtroom.cpp (hunk 1 above)
Comment (Crystalwarrior) [reply src/courtroom.cpp:4322]: 🤔 💭

Comment (stonedDiscord) [reply src/courtroom.cpp]: i have adressed this

**Review by stonedDiscord — APPROVED** [review]
average courtroom.cpp spaghetti

**Pull request merged**